docs/docs/guide/04-blog/00-build-blog.md

```diff
@@ -6,534 +6,302 @@ slug: /guide/blog
+with the ability to perform CRUD (create, read, update, and delete) operations
+on these posts. This will allow users to create new posts, read existing posts,
+update the contents of existing posts, and delete posts that are no longer
+needed.
+
+One of the features of the Ignite CLI is the ability to generate code that
+implements basic CRUD functionality. This is accomplished through the use of
+scaffolding commands, which can be used to quickly generate the necessary code
+for creating, reading, updating, and deleting data in your application.
+
+The Ignite CLI is capable of generating code for data that is stored in
+different types of data structures. This includes lists, which are collections
+of data indexed by an incrementing integer, maps, which are collections indexed
+by a custom key, and singles, which are single instances of data. By using these
+different data structures, you can customize your application to fit your
+specific needs. For example, if you are building a blog application, you may
+want to use a list to store all posts, with each post indexed by an integer.
+Alternatively, you could use a map to index each post by its unique title, or a
+single to store a single post. The choice of data structure will depend on the
+specific requirements of your application.
+
+In addition to the data structure you choose, the Ignite CLI also requires you
+to provide the name of the type of data that it will generate code for, as well
+as fields that describe the type of data. For example, if you are creating a
+blog application, you may want to create a type called "Post" that has fields
+for the "title" and "body" of the post. The Ignite CLI will use this information
+to generate the necessary code for creating, reading, updating, and deleting
+data of this type in your application.
 
-```bash
-ignite scaffold message createPost title body
 ```
-
-The `message` command accepts message name (`createPost`) and a list of fields
-(`title` and `body`) as arguments.
-
-The `message` command has created and modified several files:
-
-```
-modify proto/blog/blog/tx.proto
-modify x/blog/client/cli/tx.go
-create x/blog/client/cli/tx_create_post.go
-create x/blog/keeper/msg_server_create_post.go
-modify x/blog/module_simulation.go
-create x/blog/simulation/create_post.go
-modify x/blog/types/codec.go
-create x/blog/types/message_create_post.go
-create x/blog/types/message_create_post_test.go
-
-🎉 Created a message `createPost`.
-```
-
-As always, start with a proto file. Inside the `proto/blog/blog/tx.proto` file,
-the `MsgCreatePost` message has been created. Edit the file to add the line that
-defines the `id` for `message MsgCreatePostResponse`:
-
-```protobuf title="proto/blog/blog/tx.proto"
-message MsgCreatePost {
-  string creator = 1;
-  string title = 2;
-  string body = 3;
-}
-
-message MsgCreatePostResponse {
-  // highlight-next-line
-  uint64 id = 1;
-}
+ignite scaffold list post title body
```

Comment:
Shouldn't we mention `cd blog` first ?

Reply:
https://github.com/ignite/cli/pull/3234/commits/1d489f3d6aaa0f2225dac3e4713d5d08e1228e6a